.github/workflows/conformance.yml

```diff
@@ -109,12 +109,13 @@ jobs:
         working-directory: "gateway-api/conformance"
         run: go test -v -timeout 10m ./ --gateway-class consul-api-gateway
 
-      # TODO Waiting for webhook approval
-      # - name: Report Status
-      #   if: always()
-      #   uses: ravsamhq/notify-slack-action@v1
-      #   with:
-      #     status: ${{ job.status }}
-      #     notify_when: 'failure'
-      #   env:
-      #     SLACK_WEBHOOK_URL: ${{ secrets.ACTION_MONITORING_SLACK }}
+      - name: Report Status
+        if: always()
+        uses: ravsamhq/notify-slack-action@v1
+        with:
+          status: ${{ job.status }}
+          mention_groups: '!here'
```

Comment:
Is this supposed to be an `@here`? Also, this could potentially get pretty noisy (even though we're not planning on running these often)... do we really want to `@` anyone?

Reply:
I'd be fine with that so long as our conformance test runs aren't flaky 😅 -- I suppose we can always tune it down if it gets too noisy